releasenotes/notes/remove-importlib_metadata-dependency-a0f8138ae307acfe.yaml

```
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    Remove importlib_metadata dependency as it is no longer needed as dd-trace-py supports Python 3.8+.
```

Something like this better aligns with how we write release notes:

```suggestion
    dependencies: Remove importlib_metadata dependency since it is no longer needed. Support for Python 3.7 has been dropped, and dd-trace-py now requires Python 3.8 or later, which includes importlib.metadata in the standard library.
```